examples/framework/preferences/minimalEditor/schemas/primary.js

```diff
@@ -0,0 +1,37 @@
+ /*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+   /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    fluid.defaults("minEditor.primarySchema", {
```

Comment:
Anastasia's new way of defining the primary schema looks good to me. However, I'm not sure if it should be recommended as another way of the best practice in our documentation, just because in the preferences framework, the starter primary schema for each preference is defined individually - https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/StarterSchemas.js#L165-L248. I think one advantage of supporting defining them individually is those definitions can be scattered but still collected. If that's the only reason, Anastasia's new way seems OK to be recommended. But I don't rmb if there are other reasons.

@amb26, any thoughts?